### PR TITLE
Lesson1: Task2: Copying and archiving working directory

### PR DESCRIPTION
**Pull request by VladFear**
PR is showing work with Bash by copying directory from one place to another and archiving it.

### CODE REVIEW

**Comment by VladFear** [comment]
@StanislavGoncharovGL 
Can you check this PR?

File: 01-Git_Bash_Make/compress.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+path=$(pwd)
+cp -r . /tmp/guessanumber
+cd /tmp
+tar -zcvf guessanumber.tar.gz guessanumber/
+mkdir -p "$path/../release"
+cp guessanumber.tar.gz "$path/../release"
```
Comment (StanislavGoncharovGL) [inline 01-Git_Bash_Make/compress.sh]: I think it will be better to use round brackets when using 'path' variable..

**Comment by VladFear** [comment]
@StanislavGoncharovGL 
Added round brackets. Please check is it okay for you?

File: 01-Git_Bash_Make/compress.sh (hunk 1 above)
Comment (StanislavGoncharovGL) [reply 01-Git_Bash_Make/compress.sh]: Looks Ok

**Comment by VladFear** [comment]
@AleksandrBulyshchenko 
Can you review this PR please?

**Pull request merged**